letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -287,6 +287,10 @@ BootstrapRpmCommon() {
 
   if ! $SUDO $tool list *virtualenv >/dev/null 2>&1; then
     echo "To use Certbot, packages from the EPEL repository need to be installed."
+    if ! $SUDO $tool list epel-release >/dev/null 2>&1; then
```

Comment:
As long as we're sure that this package will *always* be visible on RPM-based systems where `certbot-auto` works.